assets/prototype/domain/eventEditor/containers/mutations/datetimes/useOnCreateDatetime.js

```diff
@@ -0,0 +1,42 @@
+import useRelations from '../../../../../infrastructure/services/relations/useRelations';
+
+import updateTicketCache from './updateTicketCache';
+import useUpdateDatetmeCache from './useUpdateDatetmeCache';
+
+const useOnCreateDatetime = () => {
+	const { updateRelations, addRelation } = useRelations();
+
+	const updateDatetmeCache = useUpdateDatetmeCache();
+
+	const onCreateDatetime = ({ proxy, datetimes, datetime, tickets }) => {
+		if (datetime.id) {
+			const datetimeIn = datetimes.nodes.map(({ id }) => id);
```

Comment:
will `nodes` property always be there?

Reply:
That's a good question. Need to handle the uneven situation as well.